--- a/build.conf
+++ b/build.conf
@@ -18,7 +18,7 @@ TYPE = standard
 # If set, this will be used as the policy
 # name.  Otherwise the policy type will be
 # used for the name.
-NAME = refpolicy
+NAME = targeted-fedora
 
 # Distribution
 # Some distributions have portions of policy
@@ -35,7 +35,7 @@ DISTRO = redhat
 # can either be allowed, denied, or the policy loading
 # can be rejected.
 # allow, deny, and reject are current options.
-#UNK_PERMS = deny
+UNK_PERMS = allow
 
 # Direct admin init
 # Setting this will allow sysadm to directly
