--- a/build.conf
+++ b/build.conf
@@ -18,7 +18,7 @@
 # If set, this will be used as the policy
 # name.  Otherwise the policy type will be
 # used for the name.
-NAME = refpolicy
+NAME = targeted-fedora
 
 # Distribution
 # Some distributions have portions of policy
@@ -27,6 +27,7 @@
 # for the distribution.
 # redhat, gentoo, debian, suse, and rhel4 are current options.
 # Fedora users should enable redhat.
+# Debian packaging: Keep redhat mode
 DISTRO = redhat
 
 # Unknown Permissions Handling
